packages/lesswrong/server/scripts/generativeModels/autoSpotlight.ts

```diff
@@ -169,4 +170,61 @@ async function createSpotlights() {
   console.log("Done creating spotlights for review winners");
 }
 
+
+const updateOldSpotlights = async () => {
+  const reviewWinners = REVIEW_WINNER_CACHE.reviewWinners
+  const postIds = reviewWinners.map(winner => winner._id);
+  const spotlights = await Spotlights.find({ documentId: { $in: postIds }, customSubtitle: null, draft: false, deletedDraft: false }).fetch();
+
+  const results = await Promise.all(spotlights.map(async (spotlight) => {
+    const reviewWinner = reviewWinners.find(reviewWinner => reviewWinner._id === spotlight.documentId);
+    return Spotlights.rawUpdateOne({_id: spotlight._id}, {$set: {customSubtitle: `Best of LessWrong ${reviewWinner?.reviewWinner.reviewYear}`}})
+  }));
+}
+
+const updateSpotlightUrlsAndPostCustomHighlights = async () => {
```

Comment:
Function name as above - the only way to know what it's updating is by reviewing the code. Could potentailly just be a doc comment instead, but something more is definitely needed here.